test-js/integration.js

```diff
@@ -628,6 +662,25 @@ describe("integration tests", function() {
       yield setupPackage(temp, sh);
 
       yield fs.writeFile(path.join(temp, "bower.json"), JSON.stringify(testBowerJson));
+
+      const psVer = yield getPursVersion(sh, assert);
+      if (semver.gte(psVer, semver.parse('0.15.0'))) {
+        // For tests to pass on the `v0.15.0-alpha-01` purs version,
+        // we need to use a custom fork of a non-core repo
+        // (i.e. working-group-purescript-es/purescript-prelude#es-modules-libraries).
+        // Since those repos' `bower.json` files point to dependencies
+        // whose "versions" are branch names as well, the `bower.json` JSON parsing fails.
+        // So, we need to remove the `bower_components` folder that gets set up
+        // when `pulp init` is called, and re-install the bower deps
+        // using the `bower.json` file.
+        //
+        // Furthermore, we have to install all 3 deps used in a typical `pulp init`
+        // setup. Otherwise, `pulp version` fails due to other dependency-related reasons.
```

Comment:
What should happen once we no longer have `#es-modules-libraries` branches?

Reply:
It's an unfortunate a cyclical dependency. When `bower install` is run by `pulp init`, it downloads the repos and stores their `bower.json` file as `.bower.json`. At some point, this file is parsed as a bower file. However, that parsing only works if the version for a dependency follows the typical semver variant (e.g. `"purescript-prelude": "^7.0.0"`), not the repo-branch variant (e.g. `"purescript-prelude": "working-group-purescript-es/purescript-prelude#es-modules-libraries"`).

So, we'd need to make a release for `prelude`, `effect`, `console`, and `psci-support` where their `bower.json` file's dependencies versions point to actual versions (e.g. `^7.0.0`), update `Init.purs` to install those versions rather than the `es-modules-libraries` branch, and then drop this code.

Reply:
OK. Then when merging this we should create an issue to come back and fix this, because it'll break later.

Reply:
I opened https://github.com/purescript-contrib/pulp/issues/402